en/docs/developer-portal/manage-application.md

```diff
@@ -8,46 +8,150 @@ Let's get started!
 
 ## Create an application
 
-1. Sign in to Choreo Developer Portal at [https://devportal.choreo.dev](https://devportal.choreo.dev). Alternatively, you can click on the Developer Portal link on the Choreo console header. 
+1. Sign in to Choreo Developer Portal at [https://devportal.choreo.dev](https://devportal.choreo.dev). Alternatively, click the Developer Portal link on the Choreo console header. 
 
     ![Developer Portal](../assets/img/developer-portal/manage-applications/developer-portal.png){.cInlineImage-half}
     
-2. Click **Applications** and then, click **+Create**.
-3. Enter the application name, select the usage policy, and optionally, add the application description. Click **Create**.
+2. Click **Applications** and click **+Create**.
+3. Enter the application name and select the usage policy. You can optionally also add the application description. Click **Create**.
 4. You are redirected to the application overview page. You can view the throttling tier, the token type, workflow status, and the application owner of the API. 
 
 ## Generate Keys
```

Comment:
We would need to change level 2 headings to follow sentence-style capitalization [1]. 
[1] https://docs.google.com/document/d/1C6t7fRbr2rXLzvdrv1wZLJkfqFY8MWR1SAzsUWoNtj0/edit#heading=h.eo1xbveo4k8a

Reply:
done